slsa/buildtype.go

```diff
@@ -53,39 +53,45 @@ type BuildType interface {
 
 // GithubActionsBuild is a basic build type for builders running in GitHub Actions.
 type GithubActionsBuild struct {
-	Context github.WorkflowContext
-	Clients ClientProvider
-	subject []intoto.Subject
+	Context  github.WorkflowContext
```

Comment:
We should maybe rename this to `WorkflowContext`

Reply:
I mostly agree and I don't think anyone's using it but since it's a public interface I'd rather not change it unless there's a better reason.

Reply:
Then I think we should have comments here to explain each the properties

Reply:
Done.